### ISSUE TITLE
Look into Fred firmware

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1702]
We're not sure if Fred's firmware can be upgraded, but we would like to see if we can do so and perhaps make him trim jump stitches.

Jackie and I will look into this and see what can be seen.

**Comment by Knottyashcreative** [comment]
Installed latest firmware and no improvements detected i.e. still not cutting the thread.  More investigations are to follow.

**Comment by Knottyashcreative** [comment]
I have read the manual and it states that the NV800E will only trim at the end of a run and stop.  This machine appears not to support multiple trimming as we had hoped.  


![automatic cutting](https://user-images.githubusercontent.com/58068720/183623294-22a15ae1-7c71-4cbe-91c2-1bdf2960d475.png)



I have researched and experimented with Hatch, Inkstitch, Embrid, and Embroidery reader to add trim commands and various methods and have not had any success.

I have attempted to use converts to look at the code within the PES file.  The PES file at this time was not outputted to a usable format i.e. usable hex code, g-code, or ASCII text.


I do however leave this baton with the next willing person.
